pkgs/top-level/stage.nix

```diff
@@ -314,7 +314,7 @@ let
       crossSystem = {
         isStatic = true;
         config = lib.systems.parse.tripleFromSystem (
-          if stdenv.hostPlatform.isLinux then
+          if stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isMlibc then
```

Comment:
Would probably make more sense for this to check for GNU, or we'll be excepting every new libc from it.

Reply:
I don't think uclibc properly supports anything in Nixpkgs, but AIUI it's mostly a specific quirk of glibc that makes it bad at static linking. @RossComputerGuy will static linking work fine with llvm-libc?

Reply:
> will static linking work fine with llvm-libc?

Yes, it's static only